### PR TITLE
:sparkles: Ask author's linkedin username (fixes #170)

### LINKED ISSUE #170: Ask for linkedin username

### PR DESCRIPTION
**Pull request by anku255**
Hi! 

This PR fixes #170.

I was confused about the emoji for LinkedIn (shown while asking for username). I ended up going with  🕴instead of 👤 but I am not sure about it. What do you think about it? 

Let me know of any changes if required. 

Thank you.

### CODE REVIEW

**Comment by kefranabg** [comment]
Nice job @anku255! What do you think of using  💼 ?

File: src/questions/author-linkedin.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const { cleanSocialNetworkUsername } = require('../utils')
+
+module.exports = () => ({
+  type: 'input',
+  message: '🕴  LinkedIn username (use empty value to skip)',
```
Comment (kefranabg) [inline src/questions/author-linkedin.js]:
```suggestion
  message: '💼  LinkedIn username (use empty value to skip)',
```

Comment (anku255) [reply src/questions/author-linkedin.js]: > Nice job @anku255! What do you think of using 💼 ?

Yes, I also thought about using this emoji. 👍

File: src/questions/author-linkedin.spec.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const askAuthorLinkedIn = require('./author-linkedin')
+
+describe('askAuthorLinkedIn', () => {
+  it('should return correct question format', () => {
+    const result = askAuthorLinkedIn()
+
+    expect(result).toEqual({
+      type: 'input',
+      message: '🕴  LinkedIn username (use empty value to skip)',
```
Comment (kefranabg) [inline src/questions/author-linkedin.spec.js]:
```suggestion
      message: '💼  LinkedIn username (use empty value to skip)',
```

Sorry forgot to suggest this one too

Comment (anku255) [reply src/questions/author-linkedin.spec.js]: That's why we have tests right? 😉

**Comment by JaredHarbison** [comment]
Cool!

**Pull request merged**